### PR TITLE
Elaborate on settings dialog help (what's this)

### PR DESCRIPTION
**Pull request by nbolton**
- Adds two _What's This?_ attributes related to the Windows service
- Rewords the related tooltips to provide a brief explanation

### CODE REVIEW

**Comment by nbolton** [comment]
@sithlord48 Maybe you can help. I'm a bit confused about the behavior of what's this... On Windows, it works for checkbox on Windows but doesn't work for combo box (but that's working fine on Linux). Something to do with Qt versions?

File: src/lib/gui/dialogs/SettingsDialogBase.ui [hunk 1]
```diff
@@ -624,8 +624,15 @@
                 <height>0</height>
                </size>
               </property>
-              <property name="toolTip">
```
[the remark this answers is not in the document]
Comment (sithlord48) [reply src/lib/gui/dialogs/SettingsDialogBase.ui:627]: they are different things

the tooltip is a normal tool tip 
the whats this is a special thing that you can see when in "what's this " mode

On linux we can generally ask the window manager to put the button in our title bar but the window manager can choose to ignore those hints and that also does not work on windows or mac os. so what you generally end up doing that enters "What's This" mode.

[the remark this answers is not in the document]
Comment (nbolton) [reply src/lib/gui/dialogs/SettingsDialogBase.ui:627]: Yes, I read the docs and my takeaway was:
- Tooltip is for a brief explanation
- What's This should expand on the tooltip and provide a detailed explanation

Do you recommend having both a tooltip and a _What's This_?

> The tip is a short piece of text reminding the user of the widget's function
> "What's This?" help is part of an application's online help system, and provides users with information about the functionality and usage of a particular widget. "What's This?" help texts are typically longer and more detailed than [tooltips](https://doc.qt.io/qt-6/qtooltip.html), but generally provide less information than that supplied by separate help windows.

**Comment by nbolton** [comment]
I'm going to guess that this is either a quirk of Qt or a bug. Either way, I don't believe it's something I can solve, so I'm going to ignore it.

On right click (expecting _What's this_):
| | Windows | Linux |
| --- | --- | --- |
| Combo box disabled | ✅ Works | ✅ Works |
| Combo box enabled | ❌ **_Nothing happens_** | Unknown* |
| Checkbox disabled | ✅ Works | ❌ **_Nothing happens_** |
| Checkbox enabled | ✅ Works | Unknown* |

**Review by sithlord48 — APPROVED** [review]

**Pull request merged**